### ISSUE TITLE
Switch H2 for Nav with any of our page tabs

### ISSUE THREAD

**Issue by fpcorso** [PopupMaker/Popup-Maker#818]
Since we have H1 as the page title and then have H2's usually right after the tabs, I don't think H2 makes sense here in terms of page structure.

I will research more on structure/accessibility and update this issue.

**Comment by fpcorso** [comment]
Yes, these should be `nav` and not `h2`.

The newer WP pages have the tabs as `nav` as well:
![image](https://user-images.githubusercontent.com/9730040/95218915-c8de6500-07c2-11eb-8382-ff6ed03a156a.png)

On our screen, there are two `H2` elements in appropriate places already (they are hidden screen-recorder headings):
![image](https://user-images.githubusercontent.com/9730040/95219075-f6c3a980-07c2-11eb-9c07-ab8b46a79732.png)

So, for structure and accessibility, these should be `nav`.

**Comment by danieliser** [comment]
@fpcorso To be fair WP changed those in the recent past. The original tabs were taken from existing interfaces within WP admin at the time they were written so they would use core styles :).

As long as core has new styles for <nav> then perfect.
